docs/usage/helpers.mdx

```diff
@@ -0,0 +1,163 @@
+---
+name: Project description helpers
+---
+
+import Message from '../components/message'
+import { List } from 'semantic-ui-react'
+
+# Project description helpers
```

Comment:
@pepibumur Have you considered changing the `[...]Description` naming to something like `[...]Toolbox`. That's my own POV but description sends me documentation inputs rather than something that I understand is more a shared space for Project.swift files. I feel that suffixes like `Core`, `Toolkit`, `Factory` or `Toolbox` would bring a better definition. But that said I might be totally wrong! I'm still definitely a Tuist beginner ;)

Reply:
Btw... on this topic, I might also suggest renaming `Project.swift` files to `Tuist.swift`, `TuistFile` or `TuistProject.swift`... I feel that `Project.swift` drops the brand aspect, which is IMO important, it would also tell a complete beginner that this file is related to Tuist, without needing to ask around

Reply:
Those are good points. Answering to the first point. We made that for consistency reasons with the Swift Package Manager and other libraries. The Swift Package Manager calls the manifests target PackageDescription. The helpers idea was inspired by Rails that calls helpers to functions that extract views’ functionality.

Regarding the second point, it was a conscious a bit influenced by the Swift Package Manager. All the files and folders that are the input of Tuist have generic names so that the name of Tuist doesn’t ens up all over the place. Following your suggestion would be much better from the branding point of vie, but something that I wouldn’t reconsider at the moment. What about opening an issue to revisit before the next major release?